packages/node/src/executors/execute/execute.impl.spec.ts

```diff
@@ -83,6 +88,7 @@ describe('NodeExecuteBuilder', () => {
 
   it('should build the application and start the built file', async () => {
     for await (const event of executeExecutor(testOptions, context)) {
+      if (!('success' in event)) throw new Error('Got Unexpected event');
```

Comment:
Represents a breaking change: the type of event yielded by `executeExecutor` is now https://github.com/dpchamps/nx/pull/1/files#diff-8b1daad050dbc745646b89f91cd20c497a23f51caf7eeaddb96a517a80b8f01fR28